bin/lib/start.js

```diff
@@ -122,7 +122,11 @@ function bin (argv, server) {
     app = solid.createServer(argv, server)
   } catch (e) {
     if (e.code === 'EACCES') {
-      console.log(colors.red.bold('ERROR'), 'You need root privileges to start on this port')
+      if (e.syscall === 'mkdir') {
+        console.log(colors.red.bold('ERROR'), `You need permissions to create '${e.path}' folder`)
```

Comment:
I just wonder if we want to catch the mkdir case earlier, i.e., in createServer itself.

Reply:
@RubenVerborgh  I was gonnna, but all the console statements in the current codebase are handled in this location. But implying EACCES is only for root for all scenarios are not valid. Should we have a check here to only imply "port" if it is 80 and 443 ?

Reply:
Might get too complex probably, I think this is already a good fix. Thanks!